bigflow/workflow.py

```diff
@@ -23,24 +25,51 @@ def __init__(self,
                  workflow_id,
                  definition,
                  schedule_interval=DEFAULT_SCHEDULE_INTERVAL,
-                 start_time_expression_factory=daily_start_time):
+                 start_time_expression_factory=daily_start_time,
```

Comment:
Workflow class shouldn't have anything to do with the infrastructure (monitoring, logging, cluster setup, etc.)

I imagine that a user configures and uses logging like this:
```
import logging
from bigflow.logging import configure_logging
from ca.scorer.config import scorer_config

config = scorer_config.resolve()
configure_logging(project_id=config['project_id'])
logger = logging.getLogger(__file__)


class SomeJob:
    def run(self, runtime):
        logger.info('logging works inside a job')


logger.info('logging works also outside a job')

raise ValueError('a exceptions is also logged into the specified project dashboard')
```